src/components/Chart/RoundsChart.js

```diff
@@ -0,0 +1,162 @@
+import cx from 'classnames';
+import * as d3 from 'd3';
+import { groupBy } from 'lodash';
+import React, { useEffect, useRef, useState } from 'react';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+
+import Axis from './Axis';
+import Line from './Line';
+
+import { getPlayerInfo } from '../../data/utils';
+
+import styles from './styles/RoundsChart.module.scss';
+
+const useResizeObserver = (
+  containerRef,
+  initialSize = { height: 0, width: 0 }
+) => {
+  const [size, setSize] = useState(initialSize);
+
+  useEffect(() => {
+    const containerElem = containerRef.current;
+
+    if (!containerElem) {
+      return;
+    }
+
+    const observer = new ResizeObserver(() => {
+      setSize({
+        height: containerElem.clientHeight,
+        width: containerElem.clientWidth,
+      });
+    });
+
+    observer.observe(containerElem);
+
+    return () => observer.unobserve(containerElem);
+  }, [containerRef]);
+
+  return size;
+};
+
+const translate = (x, y) => `translate(${x}, ${y})`;
+
+const COLORS = [
+  '#23171b',
+  '#3987f9',
+  '#2ee5ae',
+  '#95fb51',
+  '#feb927',
+  '#e54813',
+  '#900c00',
+];
+const HEIGHT = 360;
+const MARGIN = {
+  bottom: 24,
+  left: 8,
+  right: 8,
+  top: 16,
+};
+
+const parseTime = d3.timeParse('%Y-%m-%d');
+
+const RoundsChart = ({ data }) => {
+  const ref = useRef(null);
+  const { height, width } = useResizeObserver(ref);
+
+  const innerHeight = height - MARGIN.top - MARGIN.bottom;
```

Comment:
I can’t tell, but is this manual CSS handling? Would it make sense to have this live in a style sheet?

Reply:
It's not CSS; the height/width values are calculated based on the size of the container.